.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: 'CI'
+# **What it does**: Runs test on the CI.
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request: {}
+
+jobs:
+  test:
+    name: "Tests"
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+          cache: yarn
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run Tests
+        run: yarn test:ember
```

Comment:
**question**

The `test:ember` script only ensures that Ember tests are passing. Do you want to create a CI job to lint files as well (by running the `lint` script)?

Reply:
Thanks for the suggestion, I added a lint job(I took it from your repo you linked) from line 14. Let me know if this works

Reply:
Yep, looks good!